apps/web/src/services/dashboards/config.ts

```diff
@@ -83,6 +89,22 @@ export const VARIABLE_TYPES = ['MANAGED', 'CUSTOM'] as const;
 export type VariableType = typeof VARIABLE_TYPES[number];
 
 
+interface DefaultOptions {
+    value: { key: string; label: string; }[];
```

Comment:
nitpick: I prefer `values` than `value` for array... 🤓 Because it is better to know that it is an array :)

Reply:
question: does `SearchDataSourceOptions` also need `values`? 🤔

Reply:
This need discussion too.
As you know, `SearchDataSourceOptions` type don't have `values`.
But I temporarily using `values`. This need discussion and refactoring.